docs/5.0/kubernetes-5.0-migration.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Kubernetes 5.0 Migration Guide
+description: How to migrate pre-5.0 Teleport clusters to new configuration
+---
+
+# Kubernetes intergartion in 5.0
+
+In release 5.0, Teleport has changed the [Kubernetes
+integration](kubernetes-ssh.md) to improve configuration and user experience.
+These changes are backwards compatible and there is no _required_ migration.
+However, to get the most out of Teleport, you should consider migrating as
+described below.
+
+The main changes in 5.0 are:
+- new `kubernetes_service` configuration section in `teleport.yaml`, decoupled
+  from `proxy_service`
+- support for multiple Kubernetes clusters per Teleport cluster (replaces the
+  need for [Trusted Clusters](admin-guide.md#trusted-clusters))
```

Comment:
We should be a bit clearer that trusted clusters are still a useful feature for Teleport in general, just that you don't _need_ to use them to access multiple Kubernetes clusters any more.

```suggestion
  need to use [Trusted Clusters](admin-guide.md#trusted-clusters) to achieve this)
```